ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/ButtonComponentStyle.kt

```diff
@@ -32,5 +32,6 @@ internal data class ButtonComponentStyle(
         }
     }
 
+    override val visible: Boolean = stackComponentStyle.visible
```

Comment:
Hmm I see... So the button component itself won't have a visibility property... Makes sense 👍